WordPressComStatsiOS/WordPressComStatsiOS.xcodeproj/xcshareddata/xcschemes/WordPressComStatsiOS.xcscheme

```diff
@@ -52,6 +52,36 @@
                ReferencedContainer = "container:WordPressComStatsiOS.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
Stats tests seem to be working correctly, I would still try to get rid of this so we don't make unnecessary changes.